group-coordinator/src/main/java/org/apache/kafka/coordinator/group/OffsetMetadataManager.java

```diff
@@ -352,7 +374,14 @@ public CoordinatorResult<OffsetCommitResponseData, Record> commitOffset(
             });
         });
 
-        return new CoordinatorResult<>(records, response);
+        CompletableFuture<Void> appendFuture = new CompletableFuture<>();
+        appendFuture.whenComplete((__, t) -> {
```

Comment:
nit: are my eyes playing tricks on me or is the _ two __?

Reply:
:) it actually is 2, and a single underscore is a reserved keyword

Reply:
Oh -- java vs scala things 😂